package.json

```diff
@@ -4,23 +4,23 @@
   "dependencies": {
     "@create-figma-plugin/ui": "^2.1.3",
     "@create-figma-plugin/utilities": "^2.1.3",
-    "gradient-parser": "^1.0.2",
-    "hex-rgb": "^5.0.0",
+    "colord": "^2.9.3",
+    "postcss-value-parser": "^4.2.0",
     "preact": "^10",
-    "rgb-hex": "^4.0.0",
     "transformation-matrix": "^2.13.0"
   },
   "devDependencies": {
     "@create-figma-plugin/build": "^2.1.3",
     "@create-figma-plugin/tsconfig": "^2.1.3",
     "@figma/plugin-typings": "1.50.0",
-    "@types/gradient-parser": "^0.1.2",
     "mathjs": "^11.2.1",
-    "typescript": "^4"
+    "typescript": "^4",
+    "vitest": "^0.24.3"
```

Comment:
There's some (biaised) reasons here: https://vitest.dev/guide/comparisons.html#jest
Mostly it's because I'm familiar with it and it worked out of the box (including typescript).
jest would be a totaly valid choice I guess, though reading the documentation on typescript setup lost me before trial.

The tests are supposed to be compatible, so going to jest should "just" be about setting it up :)